bears/c_languages/CSecurityBear.py

```diff
@@ -0,0 +1,33 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable="flawfinder",
+        output_format="regex",
+        output_regex=r'.+:(?P<line>\d+):(?P<column>\d+):\s*'
+                     r'\[(?P<severity>\d)\]\s*'
+                     r'\((?P<origin>.+)\) (?P<message>.+)',
+        severity_map={"1": RESULT_SEVERITY.INFO,
+                      "2": RESULT_SEVERITY.INFO,
+                      "3": RESULT_SEVERITY.NORMAL,
+                      "4": RESULT_SEVERITY.NORMAL,
+                      "5": RESULT_SEVERITY.MAJOR},
+        prerequisite_check_command=('flawfinder',),
+        prerequisite_check_fail_message=('Flawfinder needs to be run with '
+                                         'python2'))
```

Comment:
no worries :)

2016-07-06 16:46 GMT+01:00 Zatreanu Adrian-Gabriel <notifications@github.com
>:

> In bears/c_languages/CSecurityBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/566#discussion_r69756749>
> :
>
> > +from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
> > +
> > +
> > +@linter(executable="flawfinder",
> > +        output_format="regex",
> > +        output_regex=r'.+:(?P<line>\d+):(?P<column>\d+):\s*'
> > +                     r'\[(?P<severity>\d)\]\s*'
> > +                     r'\((?P<origin>.+)\) (?P<message>.+)',
> > +        severity_map={"1": RESULT_SEVERITY.INFO,
> > +                      "2": RESULT_SEVERITY.INFO,
> > +                      "3": RESULT_SEVERITY.NORMAL,
> > +                      "4": RESULT_SEVERITY.NORMAL,
> > +                      "5": RESULT_SEVERITY.MAJOR},
> > +        prerequisite_check_command=('flawfinder',),
> > +        prerequisite_check_fail_message=('Flawfinder needs to be run with '
> > +                                         'python2'))
>
> hate to make you repush for this, but please period at the end :D
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/566/files/6deccf51af3a1aaf7ba930b4a6efca06951ee947#r69756749>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ9xohCzrrU9TeWJk20yACBE7oqz9jfmks5qS83bgaJpZM4JF479>
> .
>